plugins/meta/portmap/README.md

```diff
@@ -39,7 +41,9 @@ look like:
                 {
                         "type": "portmap",
                         "capabilities": {"portMappings": true},
-                        "snat": false,
+                        "snat": true,
+                        "markMasqBit": 4,
```

Comment:
Prefer to use the default value here to avoid polluting bit 4 with people using it accidentally.